lib/ansible/modules/network/enos/enos_config.py

```diff
@@ -248,32 +243,35 @@ def main():
         after=dict(type='list'),
 
         match=dict(default='line', choices=['line', 'strict', 'exact', 'none']),
-        replace=dict(default='line', choices=['line', 'block', 'config']),
+        replace=dict(default='line', choices=['line', 'block']),
 
         config=dict(),
+        defaults=dict(type='bool', default=False),
```

Comment:
`DOCUMENTATION:` for `defaults:` needs adding

Remember you can use `type: bool` rather than `choices:` in DOCUMENTATION.

Reply:
Reverting the changes back.